### PR TITLE
582 regex

### PR DESCRIPTION
**Pull request by cmwylie19**
- [x] hello-pepr-config-ignore-ns
- [x] hello-pepr-regex-name
- [x] hello-pepr-regex-ns-wrong
- [x] hello-pepr-regex-ns-bounded
- [x] hello-pepr-regex-ns-all

### CODE REVIEW

File: fast.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+```bash
+rm -rf node_modules
+npm i
+# rm -rf node_modules/pepr/src
+rm -rf node_modules/pepr/dist
+cp -r ~/pepr/dist node_modules/pepr/dist
+# cp -r ~/pepr/src node_modules/pepr/src
+```
```
Comment (cmwylie19) [inline fast.md:8]:
```suggestion
```

Delete this

File: hello-pepr-config-ignored-ns/capabilities/ignored-ns.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { Capability, a, K8s, kind } from "pepr";
+
+const name = "hello-pepr-regex-name";
```
Comment (cmwylie19) [inline hello-pepr-config-ignored-ns/capabilities/ignored-ns.ts:3]:
```suggestion
const name = "hello-pepr-config-ignored-ns";
```

File: hello-pepr-regex-name/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "hello-pepr-regex-name",
+  "version": "0.0.1",
+  "description": "Pepr Feature: Module Regex Name with Ignored NS Test",
```
Comment (btlghrants) [inline hello-pepr-regex-name/package.json:4]:
```suggestion
  "description": "Pepr Feature: Module Regex Name",
```

**Review by btlghrants — APPROVED** [review]
Big one -- looks good.

**Pull request merged**